apps/gallery/index.html

```diff
@@ -105,7 +105,7 @@ <h1 id="pick-header-title" data-l10n-id="pickoneimage2"></h1>
     </section>
 
     <section role="region" id="fullscreen-view" class="skin-dark">
-      <gaia-header id="fullscreen-toolbar-header" action="back" class="large-only">
+      <gaia-header id="fullscreen-toolbar-header" action="back" class="large-only" no-font-fit>
```

Comment:
Now that I've read the rest of the patch, I suspect that all of the headers are doing font-fits even if they are never displayed like this one. So I suppose you do need to add the no-font-fit here.  Please consider adding title-start as well just to match the other ones.